indico/modules/events/papers/schemas.py

```diff
@@ -192,6 +194,10 @@ class PaperSchema(mm.Schema):
     can_judge = Function(lambda paper, ctx: paper.can_judge(ctx.get('user')))
     can_comment = Function(lambda paper, ctx: paper.can_comment(ctx.get('user'), check_state=True))
     can_review = Function(lambda paper, ctx: paper.can_review(ctx.get('user')))
+    can_submit_proceedings = Function(lambda paper, ctx: paper.contribution.can_submit_proceedings(ctx.get('user')))
+    editing_enabled = Function(
+        lambda paper, : editable_type_settings[EditableType.paper].get(paper.event, 'editing_enabled')
```

Comment:
i didn't even know you can omit an argument like this... looks super ugly IMO, so i'd go for `paper, ctx` even if `ctx` isn't used.